NZSLDict/Classes/SearchViewController.swift

```diff
@@ -477,6 +522,72 @@ class SearchViewController: UIViewController, UISearchBarDelegate, UITableViewDa
         searchResults = dict.searchHandshape(targetHandshape, location: location)
         searchTable.reloadData()
     }
+    
+    
```

Comment:
:elephant: If you add a `// MARK: blah` comment it creates a named section in the file that you can jump to in the XCode code navigator dropdown - it can be handy to group all the implementation of a particular protocol together.